quirks/unitless-length/excluded-properties-002.html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>Many properties do not support quirky-length</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://quirks.spec.whatwg.org/#the-unitless-length-quirk">
+<meta name="assert" content="quirky-length is not supported by properties not listed.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="target"></div>
+<script>
+'use strict';
+
+var properties = [
+    'background-blend-mode',
+    'background-size',
+    'box-shadow',
+    'clip-path',
+    'column-span',
+    'filter',
+    'mask',
+    'object-position',
+    'perspective-origin',
+    'text-shadow',
+    'transform-origin'
+];
+
+for (let property of properties) {
+  test(() => {
+    if (!getComputedStyle(target)[property])
```

Comment:
I think this should fail the test instead, e.g. using assert_unreached

Reply:
Now fails the test if the property is not known.